ProcessLib/Process.cpp

```diff
@@ -232,7 +232,21 @@ void Process::assembleWithJacobian(const double t, GlobalVector const& x,
 
 void Process::constructDofTable()
 {
-    // Create single component dof in every of the mesh's nodes.
+    const int specified_prosess_id = 0;
+    if (_use_monolithic_scheme)
+    {
+        constructDofTableOfSpecifiedProsessMonolithicScheme();
+
+        return;
+    }
+
+    // For staggered scheme:
+    constructDofTableOfSpecifiedProsessStaggerdScheme(specified_prosess_id);
+}
+
+void Process::constructDofTableOfSpecifiedProsessMonolithicScheme()
```

Comment:
I'd think a short name is also sufficient:
```suggestion
void Process::constructMonolithicProcessDofTable()
```

Reply:
Changed accordingly.